ignite/cmd/network_chain_publish.go

```diff
@@ -36,8 +36,40 @@ func NewNetworkChainPublish() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "publish [source-url]",
 		Short: "Publish a new chain to start a new network",
-		Args:  cobra.ExactArgs(1),
-		RunE:  networkChainPublishHandler,
+		Long: `To begin the process of launching a blockchain with Ignite, a coordinator needs
+to publish the information about a blockchain. The only required bit of
+information is URL of the source code of the blockchain.
+
+The following command publishes the information about an example blockchain:
+
+  ignite network chain publish github.com/ignite/example
+
+This command fetches the source code of the blockchain, compiles the binary,
+verifies that a blockchain can be started with the binary and publishes the
+information about the blockchain to Ignite. The command returns a integer number
+that acts as an identifier of the chain on Ignite.
+
+By publishing a blockchain on Ignite you become the "coordinator" of this
+blockchain. A coordinator is an account that has the authority to approve and
+reject validator requests, set parameters of the blockchain and trigger the
+launch of the chain.
+
+Ignite considers every chain launch as part of a "project". A project is a
```

Comment:
A chain can actually be launched without being part of a project

Reply:
Yes, but we're documenting the behavior from the user's point of view, and Ignite CLI does consider every chain published as a part of a project (either an existing one or a new one).

Reply:
We removed automatic project creation during chain publish

Reply:
https://github.com/ignite/cli/pull/2948/commits/3384d6ca9edefd3253a4fb8a0ac838027bac9d5b